common/src/main/java/org/corfudb/common/metrics/micrometer/loggingsink/InfluxLineProtocolLoggingSink.java

```diff
@@ -20,6 +21,7 @@
 
     @NonNull
     private final Logger logger;
+    @Getter
```

Comment:
Also, in `LineTransformer` instead of returning return an `Optional.empty()`  we should fail. Right now we don't have any metrics that we don't expect to transform. Returning an empty optional can hide incorrect regexes which makes it harder to debug (i.e., https://github.com/CorfuDB/CorfuDB/pull/2825#discussion_r523245312 )